vMenu/FunctionsController.cs

```diff
@@ -2262,7 +2263,7 @@ private async Task PlayerBlipsControl()
                                             blip = AddBlipForEntity(ped);
                                         }
                                         // only manage the blip for this player if the player is nearby
-                                        if (p.Character.Position.DistanceToSquared2D(Game.PlayerPed.Position) < 500000 || Game.IsPaused)
+                                        if (minimapEnabled && (p.Character.Position.DistanceToSquared2D(Game.PlayerPed.Position) < 500000 || Game.IsPaused))
```

Comment:
So what about the normal pause menu map for people without the minimap permissions? Because basically the 'regular' blips permission is now just ignored and you just added an additional permission that does exactly the same as the old one?